persistence/trees/atomic_test.go

```diff
@@ -0,0 +1,69 @@
+package trees
+
+import (
+	"encoding/hex"
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	mock_types "github.com/pokt-network/pocket/persistence/types/mocks"
+	mockModules "github.com/pokt-network/pocket/shared/modules/mocks"
+	"github.com/rs/zerolog"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestTreeStore_AtomicUpdates(t *testing.T) {
+	ctrl := gomock.NewController(t)
+
+	mockTxIndexer := mock_types.NewMockTxIndexer(ctrl)
+	mockBus := mockModules.NewMockBus(ctrl)
+	mockPersistenceMod := mockModules.NewMockPersistenceModule(ctrl)
+
+	mockBus.EXPECT().GetPersistenceModule().AnyTimes().Return(mockPersistenceMod)
+	mockPersistenceMod.EXPECT().GetTxIndexer().AnyTimes().Return(mockTxIndexer)
+
+	ts := &treeStore{
+		logger:       &zerolog.Logger{},
```

Comment:
I don't think it is necessary, I only did it because I saw it done in a test somewhere else that just wanted a quick logger implementation for testing. What should our general recommendation be for loggers during testing?

Reply:
I made it use the bus instead 👍